server/index.js

```diff
@@ -64,8 +66,62 @@ app.get(
     res.redirect('/');
   }
 );
-app.use(errorMiddleware);
-app.use(staticMiddleware);
+
+app.get('/spotify/search/:track/:artist', (req, res) => {
+  const { track, artist } = req.params;
+  let songId = {};
+  const Spotify = require('node-spotify-api');
```

Comment:
We usually do all of our `require()` calls at the top of the file so that they don't run over and over for every request.